### PR TITLE
[VDEVOPS-529] podSecurityContext enabled by default

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: stable/deployment/values.yaml [hunk 1]
```diff
@@ -54,13 +54,13 @@ replicaCount: 1
 commonLabels: {}
 ## @param commonAnnotations Annotations to add to all deployed objects
 ##
-commonAnnotations: {}
+commonAnnotations: { foo: bar }
```
Comment (ADobrodey) [inline stable/deployment/values.yaml]: Please remove

File: stable/deployment/values.yaml [hunk 2]
```diff
@@ -73,7 +73,25 @@ priorityClassName: ""
 ##   runAsNonRoot: true
 ##   fsGroup: 1000
 ##
-podSecurityContext: { }
+podSecurityContext:
+  ## @param podSecurityContext.allowPrivilegeEscalation Controls whether a process can gain more privileges than its parent process.
+  ##
+  allowPrivilegeEscalation: false
```
Comment (ADobrodey) [inline stable/deployment/values.yaml]: is only on container level

File: stable/deployment/values.yaml [hunk 3]
```diff
@@ -73,7 +73,25 @@ priorityClassName: ""
...
+  ## @param podSecurityContext.readOnlyRootFilesystem Mounts the container's root filesystem as read-only.
+  ##
+  readOnlyRootFilesystem: true
```
Comment (ADobrodey) [inline stable/deployment/values.yaml]: is only on container level

File: stable/deployment/values.yaml (hunk 1 above)
Comment (ADobrodey) [reply stable/deployment/values.yaml]: There shouldn't be examples in default values

**Review by ADobrodey — CHANGES_REQUESTED** [review]

File: stable/deployment/values.yaml (hunk 2 above)
Comment (ghost) [reply stable/deployment/values.yaml]: removed

File: stable/deployment/values.yaml (hunk 3 above)
Comment (ghost) [reply stable/deployment/values.yaml]: removed

File: stable/deployment/values.yaml (hunk 1 above)
Comment (ghost) [reply stable/deployment/values.yaml]: replaced with {}

**Pull request closed without merging**